tests/it/language/testdata/java_11/with_package.java

```diff
@@ -0,0 +1,49 @@
+package com.example.MyApp;
+
+import java.util.HashMap;
+import java.util.logging.Logger;
+
+// Simpler form of smoke test, but with a non-default package.
+public class AppFunctions {
```

Comment:
We should add following, 
- https://docs.oracle.com/javase/1.5.0/docs/guide/language/annotations.html
- `interface` syntax
- `extends` syntax

Reading with https://tree-sitter.github.io/tree-sitter/playground, and code I know that they won't matter, but having it in test, would remove ambiguity for future readers.

Reply:
I'm going to leave this for now because I'm short on time, but please feel free to add this or create a ticket to add it when I get back!